appeals/api/src/database/seed/data-samples.js

```diff
@@ -251,6 +266,16 @@ export const appellantCaseList = {
 		originalDevelopmentDescription: 'lorem ipsum',
 		changedDevelopmentDescription: false,
 		isGreenBelt: randomBool()
+	},
+	[APPEAL_TYPE_SHORTHAND_FPA]: {
```

Comment:
See above

Reply:
As mentioned above, this would entail refactoring that is probably outside the scope of this PR